src/features/update/update-model.spec.ts

```diff
@@ -342,4 +352,105 @@ describe('UpdateModelCommand', () => {
         };
         compare(expected, newModel);
     });
+
+    it('morphs edge', () => {
+        const edgeId = 'edge'
+        const edgeRouterRegistry = createEdgeRouterRegistry();
+        context.root = graphFactory.createRoot(
+            newModelWithEdge(edgeId, [{ x: 64, y: 0 }, { x: 64, y: 128 }]));
+            // connects node1 left with node2 right at respective midpoints
+        const command1 = new TestUpdateModelCommand({
+            kind: UpdateModelCommand.KIND,
+            animate: false,
+            newRoot: newModelWithEdge(edgeId, [{ x: 136, y: 0 }])
+        }, edgeRouterRegistry);
+        const animation1 = command1.testAnimation(context.root, context);
+        expect(animation1).to.be.instanceof(MorphEdgesAnimation);
+        if (animation1 instanceof Animation) {
+            animation1.tween(0, context);
+            const edge = context.root.index.getById(edgeId) as SEdge;
+            expect(edge.routingPoints).to.be.eql([
+                { x: 64, y: 0 },
+                { x: 64, y: 128 }
+            ]);
+            animation1.tween(0.5, context);
+            expect(edge.routingPoints).to.be.eql([
+                { x: 100, y: 0 }, // midpoint between 64:0 and 136:0
+                { x: 100, y: 94 }  // midpoint between 64:128 and the interpolated 136:60
+            ]);
```

Comment:
I get this output when running `yarn test` on Gitpod:
```
  1) UpdateModelCommand
       morphs edge:

      AssertionError: expected [ { x: 64, y: 0 }, { x: 64, y: 128 } ] to deeply equal [ { x: 100, y: 0 }, { x: 100, y: 94 } ]
      + expected - actual

       [
         {
      -    "x": 64
      +    "x": 100
           "y": 0
         }
         {
      -    "x": 64
      -    "y": 128
      +    "x": 100
      +    "y": 94
         }
       ]
      
      at Context.<anonymous> (src/features/update/update-model.spec.ts:377:46)
```

Reply:
Fixed